runbotics/runbotics-orchestrator-ui/src/views/process/ProcessBuildView/Modeler/ConfigureActionPanel/widgets/TypographyWidget.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FC } from 'react';
+
+import { InfoOutlined } from '@mui/icons-material';
+import { Box, Typography, TypographyProps } from '@mui/material';
+import { WidgetProps } from '@rjsf/core';
+
+import If from 'src/components/utils/If';
+
+interface TypographyWidgetProps extends WidgetProps {
+    options: {
+        text: string;
+        variant: TypographyProps['variant'];
+        infoIcon: boolean;
+    }
+}
+
+const TypographyWidget: FC<TypographyWidgetProps> = ({ options }) => (
+    <Box display="flex" gap="5px" alignItems="center">
+        <If condition={options.infoIcon}>
+            <InfoOutlined sx={{
+                color: (theme) => theme.palette.action.active,
```

Comment:
`action.active` is meant for other purposes

use one from the grey object
![image](https://user-images.githubusercontent.com/44121335/203571332-e58013c9-9020-4b56-b472-d8a43e654e17.png)